wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,168 @@
+!roll <number>
 ```
 
-### STATS
+`!roll` draws a random number from 1 to the selected number (default is 100). An example of sending this command is shown below:
 
 ```
-!STATS <user>
+13:00 pippi: !roll 1000
+13:00 BanchoBot: pippi rolls 109 point(s)
 ```
 
-Shows the entered user's stats. The resulting stats depend on the mode they are playing or last played.
+### User statistics
 
-*Note: BanchoBot will not tell you which game mode the stats came from.*
-
-*Notice: BanchoBot will respond with `User not found` if the user has never played a game, even if the user does exist.*
-
-#### Usage Examples
+*Note: all user arguments are case-insensitive, as usernames are case-insensitive.*
 
 ```
-pippi: !stats peppy
-BanchoBot: Stats for peppy:
-BanchoBot: Store: 422,187,979 (#94718)
-BanchoBot: Plays: 7149 (lv65)
-BanchoBot: Accuracy: 87.29%
+!stats <user>
 ```
 
+`!stats` displays the entered user's user statistics (stats). The resulting stats depend on the [game mode](/wiki/Game_Modes) that the entered user is playing (or last played). Although BanchoBot will not display which game mode the stats came from. An example of sending this command is shown below:
+
+*Note: BanchoBot will respond with `User not found` if the user has never played a game, even if the user does exist.*
+
 ```
-pippi: !stats Ephemeral:
-BanchoBot: Stats for Ephemeral is Playing:
-BanchoBot: Store: 2,384,155,536 (#33697)
-BanchoBot: Plays: 14054 (lv96)
-BanchoBot: Accuracy: 94.19%
+13:01 pippi: !stats peppy
+13:01 BanchoBot: Stats for peppy:
+13:01 BanchoBot: Store: 427,514,691 (#94718)
+13:01 BanchoBot: Plays: 7348 (lv66)
+13:01 BanchoBot: Accuracy: 87.13%
 ```
 
-### WHERE
+### Where
 
 ```
-!WHERE <user>
+!where <user>
 ```
 
-Shows the entered user's current location.
+`!where` shows the entered user's current location. An example of sending this command is shown below:
 
 *Note: By default, this only shows the user's country. If the user has `Share your city location with others` enabled, it may also display their city.*
 
-#### Usage Example
-
 ```
-pippi: !where Ephemeral
-BanchoBot: Ephemeral is in Australia
+13:02 pippi: !where Ephemeral
+13:02 BanchoBot: Ephemeral is in Australia
 ```
 
 ### FAQ
 
 ```
-!FAQ <entry>
-!FAQ list
+!faq <entry>
+```
+
+```
+!faq list
 ```
 
-Shows the message of the entry. You can use the `list` argument to show all available entries. When a moderator uses this in a public channel, the response will be outputted to that channel. When normal users use this in a public channel, the response will be outputted through BanchoBot's PM.
+`!faq` shows the message of the entry. Users may use the `list` argument to show all available entries. When a moderator uses this in a public chat channel, the response will be outputted to that channel. When normal users use this in a public channel, the response will be outputted through BanchoBot's private message tab. Examples of sending this command as a normal user are shown below:
```

Comment:
- *message* -> *contents*?
- public/non-public output explanation is excessive, as it's mentioned in the beginning of the article
- it's worth mentioning you can get the article in another language if you prefix it with the language's two-letter code:
```
14:15:18 <TicClick> faq ru:lines
14:15:18 <BanchoBot> Умещайте свои мысли в меньшее количество строк, чтобы не получить сайленс.
```